AI/build.gradle

```diff
@@ -38,5 +38,13 @@ dependencies {
     implementation 'androidx.work:work-runtime-ktx:2.7.1'
     implementation 'io.insert-koin:koin-androidx-workmanager:3.3.3'
 
+    implementation 'com.android.support:appcompat-v7:28.0.0' 
+    implementation 'com.android.support:design:28.0.0' 
+    implementation 'com.android.support:support-v4:28.0.0' 
+    implementation 'org.tensorflow:tensorflow-lite:2.6.0' 
+
+    // NNAPI dependency
+    implementation 'com.android.support:nnapi:1.0.0' 
```

Comment:
It isn't correct way of implementation this library. You should read this documentation https://github.com/android/ndk-samples/tree/main/nn-samples

Reply:
Changed according to your tips